src/polyfills/index.js

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
```

Comment:
Is there a compelling reason to switch to the annotation approach? We tend to avoid annotations for edge cases in other scenarios and it seems more beneficial to be able to quickly pinpoint all the ignored files from one location. Could go either way on this one but wondering the rationale for the change @seyilonge-nci

Reply:
Felt the annotation approach better in instances like this, particularly since these are files mostly copied from the playground as the exclusion would carry over without any further configuration to avoid it being missed out.